traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -198,6 +198,29 @@ public static AuthenticatedUrlRequest emailOrderConfirmation(Order order, Sessio
                 .build(session.getToken());
     }
 
+    public static AuthenticatedUrlRequest addWishlist(List<Product> products, String travelerId, Session session, Context context) {
```

Comment:
this is actually not an add route...its a toggle route....the route and the sdk api are different.

please refer to the iOS PR and model after that

Reply:
Cool, got it, from Traveler.java's (and callback) point of view it's an add operation, just backed by router's toggle method